.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Title of the issue
+#### Issue reference:
+
+* [Title of the issue](Issue link goes here)
+
+---
+
+#### Description:
+
+* Add a description of what is the aim of this PR
```

Comment:
Could you change `PR` to `issue`?

Reply:
actually this whole thing looks like the template for a pull request...did you mean to submit a template for PRs?

I am more concerned with having a template for _issues_ than I am for pull requests. Could you change this to information that would be expected in an issue? For example, I'd like to know:

- version of `doppel-cli` you are using (semantic version of git commit hash)
- any relevant code that could be run to reproduce the issue
- description of the issue

I'll leave it to your exact discretion, but could you please make changes to make this an issue template?